pkgs/applications/networking/sync/rsync/default.nix

```diff
@@ -16,32 +15,27 @@
 , xxHash
 , enableZstd ? true
 , zstd
-, enableCopyDevicesPatch ? false
```

Comment:
Removing this option changes the calling convention of this package, which is not something we should do in stable.

Reply:
The option that the patch adds is included in this version, so we should keep the option but make it a no-op. (Also maybe warn that it's superfluous?)

Reply:
Done as suggested by @winterqt .
For further context see https://github.com/WayneD/rsync/blob/v3.2.5/NEWS.md#enhancements-1.